cmd/tikv-ctl/src/main.rs

```diff
@@ -822,41 +822,62 @@ fn flashback_whole_cluster(
     block_on(runtime.spawn(async move {
         // Prepare all regions for flashback.
         let start_ts = pd_client.get_tso().await.unwrap();
-        let mut stores_leader = load_leaders_to_each_store(&pd_client, start_key, end_key);
+        let prepare_stores_leader = Arc::new(RwLock::new(load_leaders_to_each_store(
```

Comment:
Since `try_join_all` will poll all futures one by one, it seems that the lock is unnecessary here?